command.py

```diff
@@ -0,0 +1,244 @@
+    """
+    global user_karma_list# pylint: disable=global-statement
+    user_karma_list = main_list
+
+class MemberCommands(app_commands.Group):
+    """MemberCommands Class
+
+        Its function will be registered when bot will start
+    """
+
+    def __init__(self, name: str):
+        super().__init__(name=name)
+    @app_commands.command(description="サーバーの情報を表示します")
+    async def serverinfo(self, interaction: discord.Interaction):
+        """
+        A command to show server infomation
+        """
+        guild=interaction.guild
+        guild_online_member=0
+        for online_member in guild.members:
+            if online_member.status == discord.Status.online:
+                guild_online_member =+ 1
+
+        role_list=[]
+        for role in guild.roles:
+            role_list.append(role.mention)
+        rlb=" | ".join(role_list)
+        embed=discord.Embed(title=f"{guild.name} - Info",color=discord.Color.blue(),
+                            timestamp=datetime.datetime.now())
+        embed.add_field(name="作成日",value=f"<t:{int(time.mktime(guild.created_at.timetuple()))}:R>")
+        embed.add_field(name="サーバーID",value=f"{guild.id}")
+        embed.add_field(name="所有者",value=guild.owner.mention)
+        embed.add_field(name=f"メンバー数({guild.member_count})",value=f"**{str(guild_online_member)}** オンライン\n**{guild.premium_subscription_count}** ブースター")
+        embed.add_field(name=f"チャンネル数({len(guild.channels)})",value=f"**{len(guild.text_channels)}** テキストチャンネル\n**{len(guild.voice_channels)}** ボイスチャンネル")
+        embed.add_field(name=f"ロール({len(guild.roles)})",value="".join([rlb]),inline=False)
+        embed.set_author(icon_url=interaction.user.avatar.url,name=interaction.user.global_name)
+        embed.set_footer(text=guild.name)
+
+        await interaction.response.send_message(embed=embed)
+
+    @app_commands.command(description="指定したユーザーの情報を表示します")
+    @app_commands.describe(user="情報を表示したいユーザーを指定します")
+    async def info(self, interaction: discord.Interaction, user: discord.Member = None):
+        """
+        A command to show information of a specified user
+        """
+        guild=interaction.guild
+
+        if user is None:
+            role_list=[]
+            for role in interaction.user.roles:
+                if role.name != "@everyone":
+                    role_list.append(role.mention)
+            rlb=" | ".join(role_list)
+            embed=discord.Embed(title=f"{interaction.user.name} - Info",color=discord.Color.blue(),
+                                timestamp=datetime.datetime.now())
+            embed.add_field(name="アカウント作成日",value=f"<t:{int(time.mktime(interaction.user.created_at.timetuple()))}:R> | <t:{int(time.mktime(interaction.user.created_at.timetuple()))}:D>")
+            embed.add_field(name="アカウント参加日",value=f"<t:{int(time.mktime(interaction.user.joined_at.timetuple()))}:R> | <t:{int(time.mktime(interaction.user.joined_at.timetuple()))}:D>")
```

Comment:
These lines seem too long.
```suggestion
            created_timetuple = interaction.user.created_at.timetuple()
            embed.add_field(name="アカウント作成日",
                            value=" | ".join([f"<t:{int(time.mktime(created_timetuple))}:R>",
                                              f"<t:{int(time.mktime(created_timetuple))}:D>"])
                            )
            joined_timetuple = interaction.user.joined_at.timetuple()
            embed.add_field(name="アカウント参加日",
                            value=" | ".join([f"<t:{int(time.mktime(joined_timetuple))}:R>",
                                              f"<t:{int(time.mktime(joined_timetuple))}:D>"])
                            )
```